saas/api_gateway_proxy/main.tf

```
@@ -347,6 +347,32 @@ resource "aws_wafv2_web_acl_association" "api_gateway" {
   web_acl_arn  = var.web_acl_arn
 }
 
+# Domain Configuration
+
+resource "aws_api_gateway_domain_name" "domain" {
+  certificate_arn = var.certificate_arn
+  domain_name     = var.domain_name
```

in the docs they recommend to set it in order to detect configuration drift 
```suggestion
  domain_name     = var.domain_name
  security_policy   = "TLS_1_2"
```